server/resp/pom.xml

```diff
@@ -86,6 +86,7 @@
             </configuration>
             <executions>
                <execution>
+                  <?m2e execute onConfiguration?>
```

Comment:
This just loves to sneak in :)

Reply:
still need to understand why vscode wants it for compiling in the IDE